test/e2e-common.sh

```diff
@@ -392,14 +392,20 @@ function install() {
     echo "Patch config-network to enable encryption features"
     toggle_feature system-internal-tls enabled config-network
 
-    # This is currently only supported by kourier
     if [[ "$INGRESS_CLASS" == "kourier.ingress.networking.knative.dev" ]]; then
       toggle_feature cluster-local-domain-tls enabled config-network
     fi
 
+    if [[ "$INGRESS_CLASS" == "istio.ingress.networking.knative.dev" ]]; then
+      toggle_feature cluster-local-domain-tls enabled config-network
+    fi
+
+    echo "Restart controller to enable the net-certmanager reconciler"
```

Comment:
I'd probably drop the `net-` prefix, as this is no longer a `net` thing.

Reply:
it's a `certficate` reconciler ;)